sources/build.gradle

```diff
@@ -165,6 +165,7 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.7'
     compile 'org.apache.httpcomponents:httpclient:4.5.3'
     compile 'org.apache.httpcomponents:httpmime:4.5.3'
+    compile 'org.apache.commons:commons-lang3:3.0'
```

Comment:
I'm a bit bothered to have a new library dependency for only one call

Reply:
I can remove it, but it's very common lib with common methods, and you can use it in future. And code is more readable with this lib. Are you sure, that it better to avoid this lib?

Reply:
@bcorne what are you thinking about it? Should I remove this dependency?

Reply:
I'd prefer to avoid as much as possible to change the transitive dependencies so if you could that would be nice.